pkg/database/user_stats.go

```diff
@@ -30,3 +32,14 @@ type UserStats struct {
 func (UserStats) TableName() string {
 	return "user_stats"
 }
+
+// Save saves some UserStats to the database.
+// This function is provided for testing only.
+func (u *UserStats) Save(db *Database) error {
```

Comment:
Is it too painful to call `db.SaveStats(stat)` - it's more in line with our current pattern to put the `SaveX` on the database instead of on the model.

Reply:
No, it's not too painful, but that function doesn't exist. Changed this function over to that format.